rundeckapp/grails-app/services/rundeck/services/JobMetadataService.groovy

```diff
@@ -97,7 +97,7 @@ class JobMetadataService {
      * @param id jobid
      */
     def removeJobPluginMetaAll(final String project, final String id) {
-        pluginMetaDataProvider.deleteAllByProjectAndKeyLike(project, id + '/%')
+        pluginMetaDataProvider.deleteAllByProjectAndKeyLike(project, id)
```

Comment:
I think this should be reverted, the parameter should be the LIKE query, so `id + '/%'` is correct

Reply:
@ocerda this need to be added